packages/manager/fragments/navbar/src/hamburger-menu.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { emit } from '@ovh-ux/ufrontend/communication';
+import style from './navbar.scss';
+
+function HamburgerMenu({ universe, universes }) {
+  const [opened, setOpened] = useState(false);
+  const { t } = useTranslation();
+
+  function onUniverseClick({ event, destination }) {
+    if (universe === destination) {
+      event.preventDefault();
+      emit({
+        id: 'navigation-sidebar.show',
+      });
+      setOpened(false);
+    }
+  }
+
+  return (
+    <div>
+      <button
+        className="oui-navbar-toggler oui-navbar-toggler_button"
+        onClick={() => setOpened(!opened)}
+        aria-expanded={opened}
+      >
+        <span className="oui-navbar-toggler__hamburger">
+          <span></span>
+          <span></span>
+          <span></span>
+          <span></span>
+        </span>
```

Comment:
Not yours but we should maybe ask CX to provide us with an icon for this

Reply:
yep i can replace this one with two buttons and a oui-hambuger-icon but we would lose this marvelous animation ... https://github.com/ovh/ovh-ui-kit/blob/fe3fa0f5b7cb31001e88bf1fa54575c7e3f2f265/packages/components/navbar/src/less/navbar.less#L280